internal/cli/query/query.go

```
@@ -32,25 +31,30 @@ func Command() *cobra.Command {
 }
 
-func runRaw(w io.Writer, query string) error {
+// runRaw POSTs `query` to addr's /graphql endpoint and pretty-prints
+// the JSON response. Exported via verbs that wrap it.
+func runRaw(w io.Writer, addr, query string) error {
 	body, err := json.Marshal(map[string]string{"query": query})
 	if err != nil {
 		return fmt.Errorf("marshal request: %w", err)
 	}
-	req, err := http.NewRequest(http.MethodPost, "http://"+server.DefaultAddr+"/graphql", bytes.NewReader(body))
+	req, err := http.NewRequest(http.MethodPost, "http://"+addr+"/graphql", bytes.NewReader(body))
```

golangci-lint (2.12.1)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)